_posts/2023-02-13-homebrew-4.0.0.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 4.0.0
+author: MikeMcQuaid
+redirect_from: /blog/4.0.0/
+---
+
+Today, I'd like to announce Homebrew 4.0.0. The most significant change since 3.6.0 enables significantly faster Homebrew-maintained tap updates by migrating from Git-cloned taps to JSON downloads.
+
+Major changes and deprecations since 3.6.0:
+
+- Using JSON files downloaded from [formulae.brew.sh](https://formulae.brew.sh) rather than local homebrew-core and homebrew-cask taps.
```

Comment:
```suggestion
- Relying on JSON files downloaded from [formulae.brew.sh](https://formulae.brew.sh) for package installation rather than local homebrew/core and homebrew/cask taps.
```

Reply:
@EricFromCanada mild edit as I like the rest but don't like "Relying on"